src/ydn/db/base/base.js

```diff
@@ -137,7 +137,7 @@ ydn.db.base.IDBTransaction = !ydn.db.base.OLD_INDEXEDDB_SUPPORT ? ydn.db.base.St
         ('LOADING' in goog.global.IDBRequest)) ?
         goog.global.IDBTransaction :
         // old chrome use predefined enum, it can be string or numeric. ?
-        (goog.global.webkitIDBRequest &&
```

Comment:
Yes, as you can see above in line 137, with defining `ydn.db.base.OLD_INDEXEDDB_SUPPORT` to `true`, this line is not even in compiled js. That is how, new release are compiled. However if the compiled file is to be used in Android 2.3 or old chrome browser, that warning is unavoidable.